### PR TITLE
Update graphql examples to recursively show nested fields

### PR DESCRIPTION
**Pull request by stnguyen90**
## What does this PR do?

GraphQL requests must always specify the attributes to be returned in the request. This PR ensures response models are processed recursively so that all examples display all attributes for nested models.

## Test Plan

Manually generated.

databases/list-documents.md:

```graphql
query {
    databasesListDocuments(
        databaseId: "[DATABASE_ID]",
        collectionId: "[COLLECTION_ID]"
    ) {
        total
        documents {
            _id
            _collectionId
            _databaseId
            _createdAt
            _updatedAt
            _permissions
            data
        }
    }
}
```

teams/get.md:

```graphql
query {
    teamsGet(
        teamId: "[TEAM_ID]"
    ) {
        _id
        _createdAt
        _updatedAt
        name
        total
    }
}
```

users/list.md:

```graphql
query {
    usersList {
        total
        users {
            _id
            _createdAt
            _updatedAt
            name
            password
            hash
            hashOptions
            registration
            status
            passwordUpdate
            email
            phone
            emailVerification
            phoneVerification
            prefs {
                data
            }
        }
    }
}
```

## Related PRs and Issues

* https://github.com/appwrite/sdk-generator/pull/598

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

File: docs/examples/1.2.x/client-graphql/examples/storage/list-files.md [hunk 1]
```diff
@@ -3,6 +3,18 @@ query {
         bucketId: "[BUCKET_ID]"
     ) {
         total
-        files
+        files {
+            _id
+            bucketId
```
Comment (gewenyu99) [inline docs/examples/1.2.x/client-graphql/examples/storage/list-files.md:8]: Is bucketID not underscored?

File: docs/examples/1.2.x/server-graphql/examples/storage/list-files.md [hunk 2]
```diff
@@ -3,6 +3,18 @@ query {
         bucketId: "[BUCKET_ID]"
     ) {
         total
-        files
+        files {
+            _id
+            bucketId
```
Comment (gewenyu99) [inline docs/examples/1.2.x/server-graphql/examples/storage/list-files.md:8]: Ditto

**Review by gewenyu99 — CHANGES_REQUESTED** [review]
Did we miss _bucketId?

**Review by gewenyu99 — APPROVED** [review]
nvm LGTM

**Review by eldadfux — APPROVED** [review]

**Pull request merged**